helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -623,6 +598,14 @@ public void rollingUpdateStep() {
     }
   }
 
+  private void emitEvents(final List<Map<String, Object>> events) {
```

Comment:
Maybe rename to `emitDeploymentGroupEvents`? It's using a rather specific topic after all.